pygrocy/grocy.py

```diff
@@ -324,6 +326,42 @@ def id(self) -> str:
     def name(self) -> str:
         return self._name
 
+class MealPlanItem(object):
```

Comment:
Does this represent a meal plan or a meal plan's item?

Reply:
A meal plan item. A meal plan is just a list of meal plan items, which apparently are stored as recipes in the underlying data model with day properties.